diff -ur orig/debian/control no-pandoc/debian/control
--- orig/debian/control	2024-11-15 14:30:56.339248449 +0100
+++ no-pandoc/debian/control	2024-11-15 14:33:30.198747930 +0100
@@ -26,7 +26,6 @@
  pyflakes3 <!nocheck>,
  pycodestyle <!nocheck> | pep8 <!nocheck>,
  python3-nose <!nocheck>,
- pandoc,
  openvswitch-switch [!riscv64] <!nocheck>,
 Vcs-Git: https://salsa.debian.org/debian/netplan.io.git
 Vcs-Browser: https://salsa.debian.org/debian/netplan.io
diff -ur orig/Makefile no-pandoc/Makefile
--- orig/Makefile	2024-11-15 14:31:10.783200450 +0100
+++ no-pandoc/Makefile	2024-11-15 14:33:30.198747930 +0100
@@ -50,7 +50,7 @@
 PYCODESTYLE3 ?= $(shell command -v pycodestyle-3 || command -v pycodestyle || command -v pep8 || echo true)
 NOSETESTS3 ?= $(shell command -v nosetests-3 || command -v nosetests3 || echo true)
 
-default: netplan/_features.py generate netplan-dbus dbus/io.netplan.Netplan.service doc/netplan.html doc/netplan.5 doc/netplan-generate.8 doc/netplan-apply.8 doc/netplan-try.8 doc/netplan-dbus.8 doc/netplan-get.8 doc/netplan-set.8
+default: netplan/_features.py generate netplan-dbus dbus/io.netplan.Netplan.service
 
 %.o: src/%.c
 	$(CC) $(BUILDFLAGS) $(CFLAGS) $(LDFLAGS) -c $^ `pkg-config --cflags --libs glib-2.0 gio-2.0 yaml-0.1 uuid`
@@ -134,10 +134,10 @@
 	install -m 644 include/*.h $(DESTDIR)/$(INCLUDEDIR)/netplan/
 	# TODO: install pkg-config once available
 	# docs, data
-	install -m 644 doc/*.html $(DESTDIR)/$(DOCDIR)/netplan/
+	#install -m 644 doc/*.html $(DESTDIR)/$(DOCDIR)/netplan/
 	install -m 644 examples/*.yaml $(DESTDIR)/$(DOCDIR)/netplan/examples/
-	install -m 644 doc/*.5 $(DESTDIR)/$(MANDIR)/man5/
-	install -m 644 doc/*.8 $(DESTDIR)/$(MANDIR)/man8/
+	#install -m 644 doc/*.5 $(DESTDIR)/$(MANDIR)/man5/
+	#install -m 644 doc/*.8 $(DESTDIR)/$(MANDIR)/man8/
 	install -T -D -m 644 netplan.completions $(DESTDIR)/$(BASH_COMPLETIONS_DIR)/netplan
 	# dbus
 	mkdir -p $(DESTDIR)/$(DATADIR)/dbus-1/system.d $(DESTDIR)/$(DATADIR)/dbus-1/system-services
@@ -149,13 +149,13 @@
 	sed -e "s#@ROOTLIBEXECDIR@#$(ROOTLIBEXECDIR)#" $< > $@
 
 
-%.html: %.md
-	pandoc -s --metadata title="Netplan reference" --toc -o $@ $<
+#%.html: %.md
+#	pandoc -s --metadata title="Netplan reference" --toc -o $@ $<
 
-doc/netplan.5: doc/manpage-header.md doc/netplan.md doc/manpage-footer.md
-	pandoc -s -o $@ $^
+#doc/netplan.5: doc/manpage-header.md doc/netplan.md doc/manpage-footer.md
+#	pandoc -s -o $@ $^
 
-%.8: %.md
-	pandoc -s -o $@ $^
+#%.8: %.md
+#	pandoc -s -o $@ $^
 
 .PHONY: clean
