diff -Nur openvswitch-2.13.5.orig/debian/changelog openvswitch-2.13.5/debian/changelog
--- openvswitch-2.13.5.orig/debian/changelog	2022-01-13 11:57:22.000000000 +0100
+++ openvswitch-2.13.5/debian/changelog	2022-08-07 14:23:25.589172898 +0200
@@ -1,3 +1,9 @@
+openvswitch (2.13.5-0dp01~jammy1) jammy; urgency=medium
+
+  * Fixed compatibility with python2
+
+ -- Uli Heller <uli.heller@daemons-point.com>  Sun, 07 Aug 2022 14:22:34 +0200
+
 openvswitch (2.13.5-0ubuntu1) focal; urgency=medium
 
   * New upstream point release (LP: #1956754).
diff -Nur openvswitch-2.13.5.orig/debian/patches/series openvswitch-2.13.5/debian/patches/series
--- openvswitch-2.13.5.orig/debian/patches/series	2022-01-13 11:55:50.000000000 +0100
+++ openvswitch-2.13.5/debian/patches/series	2022-08-07 14:27:41.212873805 +0200
@@ -1 +1,2 @@
 py3-compat.patch
+uli-python2
diff -Nur openvswitch-2.13.5.orig/debian/patches/uli-python2 openvswitch-2.13.5/debian/patches/uli-python2
--- openvswitch-2.13.5.orig/debian/patches/uli-python2	1970-01-01 01:00:00.000000000 +0100
+++ openvswitch-2.13.5/debian/patches/uli-python2	2022-08-07 14:27:45.104869251 +0200
@@ -0,0 +1,47 @@
+Description: <short summary of the patch>
+ TODO: Put a short summary on the line above and replace this paragraph
+ with a longer explanation of this change. Complete the meta-information
+ with other relevant fields (see below for details). To make it easier, the
+ information below has been extracted from the changelog. Adjust it or drop
+ it.
+ .
+ openvswitch (2.13.5-0dp01~jammy1) jammy; urgency=medium
+ .
+   * Fixed compatibility with python2
+Author: Uli Heller <uli.heller@daemons-point.com>
+
+---
+The information above should follow the Patch Tagging Guidelines, please
+checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
+are templates for supplementary fields that you might want to add:
+
+Origin: <vendor|upstream|other>, <url of original patch>
+Bug: <url in upstream bugtracker>
+Bug-Debian: https://bugs.debian.org/<bugnumber>
+Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
+Forwarded: <no|not-needed|url proving that it has been forwarded>
+Reviewed-By: <name and email of someone who approved the patch>
+Last-Update: 2022-08-07
+
+--- openvswitch-2.13.5.orig/python/setup.py
++++ openvswitch-2.13.5/python/setup.py
+@@ -24,8 +24,7 @@ try:
+     # Try to set the version from the generated ovs/version.py
+     exec(open("ovs/version.py").read())
+ except IOError:
+-    print("Ensure version.py is created by running make python/ovs/version.py",
+-          file=sys.stderr)
++    print >>sys.stderr, "Ensure version.py is created by running make python/ovs/version.py"
+     sys.exit(-1)
+ 
+ try:
+@@ -33,8 +32,7 @@ try:
+     # don't need to exec()
+     open("ovs/dirs.py")
+ except IOError:
+-    print("Ensure dirs.py is created by running make python/ovs/dirs.py",
+-          file=sys.stderr)
++    print >>sys.stderr, "Ensure dirs.py is created by running make python/ovs/dirs.py"
+     sys.exit(-1)
+ 
+ ext_errors = (CCompilerError, DistutilsExecError, DistutilsPlatformError)
